docs/myrocks/variables.md

```diff
@@ -576,6 +622,22 @@ If you disable this feature, RocksDB allocates additional memory to maintain the
 
 The variable was implemented in [Percona Server for MySQL 8.0.27-18](../release-notes/Percona-Server-8.0.27-18.md#id1). Cancels all ongoing manual compactions.
 
+### `rocksdb_charge_memory`
+
+| Option       | Description                             |
+|--------------|-----------------------------------------|
+| Command-line | --rocksdb_charge_memory                 |
+| Dynamic      | No                                      |
+| Scope        | Global                                  |
+| Data type    | Boolean                                 |
+| Default      | OFF                                     |
+
+The variable was implemented in [Percona Server for MySQL 8.0.33-25](../release-notes/Percona-Server-8.0.33-25.md).
+
+Turns on RocksDB memory-charging related features (i.e., rocksdb_charge_memory, rocksdb_use_write_buffer_manager) from `cnf` files.
```

Comment:
`Turns on RocksDB memory-charging related features (BlockBasedTableOptions::cache_usage_options.options.charged) from cnf files.`
`This option is related to rocksdb_use_write_buffer_manager.`
`This is an experimental feature that may be removed in the future.`

Reply:
Updated